dbsnap_verify/rds_funcs.py

```diff
@@ -0,0 +1,234 @@
+from operator import itemgetter
+
+from random import choice
+
+from string import (
+    letters,
+    digits,
+)
+
+VALID_SNAPSHOT_TYPES = ["automated", "manual"]
+
+
+def generate_password(size=9, pool=None):
```

Comment:
Could we build the password based on a hash of unchangeable values, that way you don't have to store the password for later runs?

Reply:
I would prefer not to use a password that is deterministic just in case the database accidentally stays alive / and or becomes misconfigured and is accessible from Internet or something. I would rather use a random password that we store for a short time in plain text then something that could be known or computed by an attacker before hand.

Reply:
I think the best solution is to allow this lambda iam role to store the plaintext password in `SSM`.